GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+		basemass = -1
+		TANK
+		{
+			name = LqdOxygen
+			amount = 620252.3135
+			maxAmount = 620252.3135
+		}
+	}
+}
+
+
+@PART[EnergiaLH2Tank6SL]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+//    @CoMOffset = 0.0, 3.55, -0.0
+	@mass = 64.4203
+	@maxTemp = 1447
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 1668501.454
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = LqdHydrogen
+			amount = 1668501.454
+			maxAmount = 1668501.454
+		}
+	}
+}
+
+
+
+@PART[EnergiaLOXTank6SL]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+//    @CoMOffset = 0.0, 3.55, -0.0
+	@mass = 12.0153
+	@maxTemp = 1973
+	%manufacturer = RKK Energia
+	@attachRules = 1,0,1,1,0
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 620252.3135
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = LqdOxygen
+			amount = 620252.3135
+			maxAmount = 620252.3135
+		}
+	}
+}
+
+
+
+
+
+
+
+
+
+//@PART[Energia_Second_Stage_6SL]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+    @CoMOffset = 0.0, 3.55, -0.0
+	@mass = 76.4356
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+
+	@attachRules = 1,1,1,1,0
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 2342353.2875
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = LqdHydrogen
+			amount = 1707575.454
+			maxAmount = 1707575.454
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 634777.8335
+			maxAmount = 634777.8335
+		}
+	}
+}
+
+//--- D4.10    L26.5
+//dry mass 1 block-a +rd0170 62.666t   mass fuel 309.85
+
+
+
+
+// 2 468 320t
+@PART[Energia_First_Stage]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	@CoMOffset = 0.0, 0.0, -0.0	
+	%scale = 1.0
+	@rescaleFactor = 1.5625
+	@mass = 42.865
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 302599.1276                  //  314511.1276
+		type = Default
+		basemass = -1
+		TANK
+		{
+			name = Kerosene
+			amount = 104699.8756        // 108820.8756
+			maxAmount = 104699.8756       
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 197899.252                // 205690.252
+			maxAmount = 197899.252
+		}
+	}
+}
+
+@PART[Energia_Central_Power_Units]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+	%scale = 1.0
+	@rescaleFactor = 1.25
+	@mass = 0.48732
+	@maxTemp = 1500
+	%manufacturer = RKK Energia
+	@attachRules = 1,1,1,1,0
+	@fuelCrossFeed = False	
+	!RESOURCE[LiquidFuel]
+	{
+	}
+	!RESOURCE[Oxidizer]
+	{
+	}

```

Comment:
These don't seem the kind of parts that should carry LF/Ox from stock.  (This is repeated in the next several parts/comments)